src/Parts/Message.php

```diff
@@ -53,19 +53,19 @@ class Message
     public ?string $application_id;
     public ?MessageReference $message_reference;
     public ?Bitwise $flags;
-    public ?Message $referenced_message;
+    public ?self $referenced_message;
     public ?MessageInteraction $interaction;
     public ?Channel $thread;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Component[]
+     * @var Component[]
      */
     public array $components;
     /**
-     * @var \Ragnarok\Fenrir\Parts\MessageStickerItem[]
+     * @var MessageStickerItem[]
      */
     public ?array $sticker_items;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Sticker[]
+     * @var Sticker[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)